Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -165,6 +236,8 @@
         "ArtifactSuccessfullyDownloaded": "Successfully downloaded artifact to %s",
         "ArtifactDownloadUrl": "Downloading artifacts from: %s",
         "DownloadingCommitsAndWorkItems": "Downloading commits and work items",
+        "ArtifactProviderNotSupported":"Artifact provider: %s is not supported.",
```

Comment:
More descriptive message will be good here "Downloading artifacts from %s is not supported". How will we get this error? Can user type in a custom provider in the picklist?

Reply:
This error can't come due to user entering values as pick list is non editable. i can very well remove this exception but had kept it just in case.